emcc

```diff
@@ -681,6 +683,16 @@ try:
       if not default_object_extension.startswith('.'):
         default_object_extension = '.' + default_object_extension
       newargs[i+1] = ''
+    elif newargs[i] == '-msse':
+      settings_changes.append('SSE1=1')
+      newargs.append('-D__SSE__=1')
+      newargs[i] = ''
+    elif newargs[i] == '-msse2':
+      settings_changes.append('SSE1=1')
+      settings_changes.append('SSE2=1')
```

Comment:
Can we just do `settings_changes.append('SIMD=1')` here, so that we don't need an explicit SSE{1,2} setting in the rest of Emscripten?